google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddResponsiveSearchAdFull.java

```diff
@@ -582,22 +536,22 @@ private static void addKeywords(
                     .setMatchType(KeywordMatchType.BROAD));
 
     // Uncomment the below line if you want to change this keyword to a negative target.
-    // keyword3.setNegative(true);
+    // keyword.setNegative(true);
 
     // Optional repeated field
-    // keyword3.setFinalUrls(0, "https://www.example.com");
+    // keyword.setFinalUrls(0, "https://www.example.com");
```

Comment:
This will throw an `IndexOutOfBoundsException`. Please replace with:

```
// keyword.addFinalUrls("https://www.example.com");
```

Note the plural `Urls` in the method name, which is a protobuf quirk.